README.md

```diff
@@ -49,7 +49,7 @@ Here is a short video from @cpw showing how to install and setup Forge
 ## Pull requests
 
 Pull requests should target the current default branch. Currently, that is 
- the 1.10.x branch for Minecraft 1.10.2.
+ the 1.11.x branch for Minecraft 1.11.2.
```

Comment:
PRs should try to do one thing only, and this fix is unrelated to anything else in the PR.